packages/framework/src/DataFixtures/DemoMultidomain/CategoryRootDataFixture.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
There is no description why is this class added

Reply:
Like in CM

Reply:
I will add it at the final rebase